addons/mail/static/src/models/thread/thread.js

```diff
@@ -1467,6 +1513,10 @@ function factory(dependencies) {
         serverFoldState: attr({
             default: 'closed',
         }),
+        /**
+         * This field containt all the suggested recipients.
```

Comment:
"this field containt" (also typo "contains") is useless information, we know it's the doc about the field.

-> "Determines suggested recipients."

A somewhat new guideline is to use "determines" as first word when changing the value has an effect, while using "states" as first word when it's purely informative (often for computed fields).